File: backend/PhotoGallery/photo_gallery/urls.py
from django.urls import path
from .views import *

urlpatterns = [
    path('user', UserApiView.as_view()),
    path('user/<id>', UserApiViewIdEndpoint.as_view()),
    path('login', LoginApiView.as_view()),
    path('album', AlbumApiView.as_view()),
    path('album/<id>', AlbumApiViewIdEndpoint.as_view()),
    path('photo', PhotoApiView.as_view()),
    path('photo/<id>', PhotoApiViewIdEndpoint.as_view()),

    path('album/like/<album_id>', like_album),
    path('album/unlike/<album_id>', unlike_album),
    path('photo/like/<photo_id>', like_photo),
    path('photo/unlike/<photo_id>', unlike_photo),
]

File: frontend/src/app/services/urls.ts
export class Urls{
    public BASE_URL : string = "http://localhost:8000";
    public ALBUM_URL : string = this.BASE_URL + "/album";
    public USER_URL : string = this.BASE_URL + "/user";
    public LOGIN_URL : string = this.BASE_URL + "/login";
    public PHOTO_URL : string = this.BASE_URL + "/photo";
    public ALBUM_LIKE_URL: string = this.BASE_URL + "/album/like"
    public ALBUM_UNLIKE_URL: string = this.BASE_URL + "/album/unlike"
    public PHOTO_LIKE_URL: string = this.BASE_URL + "/photo/like"
    public PHOTO_UNLIKE_URL: string = this.BASE_URL + "/photo/unlike"
    
    
}
